pages/services/data-science-engine/1.0.0/hdfs/index.md

```diff
@@ -132,3 +132,52 @@ You can also specify them through the UI.
                   }
     }
     ```
+   
+# TensorBoard
+
+{{ model.techName }} comes with TensorBoard installed. It can be found at
+`http://<dcos-url>/service/{{model.serviceName}}/tensorboard/`.
+
+## Log directory
+
+TensorBoard reads log data from specific directory, with the default being `/mnt/mesos/sandbox`. It can be changed
+with `advanced.tensorboard_logdir` option. HDFS paths are supported as well. 
+
+Here is an example:
+
+1. Install HDFS:
```

Comment:
The same instructions are in the Spark History section above. Can you move the `Install HDFS` section higher up the page and both sections reference the same instructions?